### PR TITLE
Added Docker file

### PR DESCRIPTION
**Pull request by sunsetdk**

### CODE REVIEW

**Comment by georgeherby** [comment]
Happy to merge if someone else can confirm this works.

Also could you please add some detail to the README for setup with docker

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Basic docker image for RocketMap
```
Comment (georgeherby) [inline Dockerfile]: These comments need updating as they are for RocketMap :)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Usage:
+#   docker build -t rocketmap .
+#   docker run -d -P rocketmap -a ptc -u YOURUSERNAME -p YOURPASSWORD -l "Seattle, WA" -st 10 --gmaps-key CHECKTHEWIKI
+
+FROM python:3.6
```
Comment (georgeherby) [inline Dockerfile]: MIght aswell run on 3.7 as in the README it says to use it (although other 3.x are fine :)

**Comment by clburlison** [comment]
I can confirm this works.

**Comment by georgeherby** [comment]
Thanks, just waiting for @sunsetdk to update the feedback I have then I will merge

**Comment by georgeherby** [comment]
@clburlison As @sunsetdk is not replying could you be able to advice what i should change the `docker build` and `docker run` to be?

**Comment by sunsetdk** [comment]
Hi
Sorry. I keep forgetting to get back. 

The build command is (run from within the folder):

docker build -t madevice  . 

The full stop at the end is on purpose. This will build an image called madevice

docker run -d -ti --name=madevice  --restart=unless-stopped madevice

will create a container from the image madevice and call the container madevice

**Pull request merged**